src/main/java/org/folio/fqm/resource/EntityTypeController.java

```diff
@@ -34,8 +34,8 @@ public ResponseEntity<List<EntityTypeSummary>> getEntityTypeSummary(List<UUID> e
   }
 
   @Override
-  public ResponseEntity<ColumnValues> getColumnValues(UUID entityTypeId, String columnName, String search) {
-    return ResponseEntity.ok(entityTypeService.getColumnValues(entityTypeId, columnName, search));
+  public ResponseEntity<ColumnValues> getColumnValues(UUID entityTypeId, String fieldName, String search) {
```

Comment:
Ideally, I'd like to rename this type to `FieldValues` and the method to `getFieldValues`, but that needs to come from folio-query-tool-metadata and will come later.

Reply:
Unfortunately, this API endpoint actually has "columns" in the path :/ I'm all for changing that, but it will be a pain (my vote would be to add a new endpoint that uses "fields" instead of "columns" and keep the old one around, then mark the old one as deprecated and have it just call the new one, then we can update the query builder at our leisure and remove the deprecated endpoint later)